src/types.d.ts

```diff
@@ -690,8 +709,3 @@ export type KVBufferStore = {
   has(key: string): Promise<boolean>;
   close(): Promise<void>;
 };
-
-export interface SignatureSource {
```

Comment:
Even though we don't require them, I'd like to keep a `SignatureSource` and `OwnerSource` interface here for documentation and also implement them on the fetchers. That makes it clear what we need to implement if we wanted to pull these from other sources in the future.

Reply:
https://github.com/ar-io/ar-io-node/pull/356/commits/381b33e48a393a50a48131067507ad05fdd16ba2

Reply:
@djwhitt Sorry! I read it wrong.
Here's the update [`381b33e` (#356)](https://github.com/ar-io/ar-io-node/pull/356/commits/381b33e48a393a50a48131067507ad05fdd16ba2)